src/spacel/provision/template/app.py

```diff
@@ -213,12 +262,28 @@ def app(self, app, region):
             }
             del params['PrivateElbSubnet01'], params['PublicElbSubnet01']
             del (resources['PublicElb'], resources['PrivateElb'],
-                 resources['ElbSg'], resources['DnsRecord'],
+                 resources['ElbSg'],
                  resources['ElbHealthPolicy'],
                  resources['Asg']['Properties']['LoadBalancerNames'])
             resources['Asg']['Properties']['HealthCheckType'] = 'EC2'
             params['ElbScheme']['Default'] = 'disabled'
 
+            # Resolve the DNS record
```

Comment:
Noice!

We _could_ tie these entries to N `Route53::HealthCheck`s too? Some day...